core/client/pom.xml

```diff
@@ -25,6 +25,7 @@
   <modules>
     <module>fs</module>
     <module>hdfs</module>
```

Comment:
Need to rename the existing one to `hdfs2`?

Reply:
i am not sure if renaming will affect backcompat